lib/commands/general.js

```diff
@@ -9,6 +9,10 @@ let extensions = {},
     commands = {},
     helpers = {};
 
+commands.active = async function () {
```

Comment:
is there an element commands file? doesn't _really_ matter where this goes but if there's an elements file that would be more sensible I think.

Reply:
Thanks for feedback. I put it to `general.js` to be consistent with `appium-ios-driver` project but I can move it to element.

Reply:
Ok I moved it to `element.js` and changed to GET.